services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -59,57 +76,120 @@ class Config(CommonConfig):
         env_prefix = "DIRECTOR_"
 
 
-class DynamicServicesSettings(BaseSettings):
-    enabled: bool = Field(True, description="Enables/Disables connection with service")
+_DYNAMIC_SIDECAR_DOCKER_IMAGE_RE = (
+    r"(^(local|itisfoundation)/)?(dynamic-sidecar):([\w]*)"
+)
 
-    class Config(CommonConfig):
-        pass
 
+class DynamicSidecarSettings(BaseSettings):
+    boot_mode: BootModeEnum = Field(
+        BootModeEnum.PRODUCTION,
+        description="Used to compute where or not should start sidecar in development mode",
+        env="SC_BOOT_MODE",
+    )
+    image: str = Field(
+        ...,
+        regex=_DYNAMIC_SIDECAR_DOCKER_IMAGE_RE,
+        description="used by the director to start a specific version of the dynamic-sidecar",
+    )
 
-class PGSettings(PostgresSettings):
-    enabled: bool = Field(True, description="Enables/Disables connection with service")
+    port: PortInt = Field(
+        8000,
+        description="port on which the webserver for the dynamic-sidecar is exposed",
+    )
+    mount_path_dev: Optional[Path] = Field(
+        None,
+        description="optional, only used for development, mounts the source of the dynamic-sidecar",
+    )
 
-    class Config(CommonConfig, PostgresSettings.Config):
-        env_prefix = "POSTGRES_"
+    expose_port: bool = Field(
+        False, description="exposes the service on localhost for debuging and testing"
+    )
+
+    simcore_services_network_name: str = Field(
+        ...,
+        regex=SERVICE_NETWORK_RE,
+        description="network all dynamic services are connected to",
+        env="SIMCORE_SERVICES_NETWORK_NAME",
+    )
+    api_request_timeout: PositiveInt = Field(
+        15,
+        description=(
+            "the default timeout each request to the dynamic-sidecar API in seconds; as per "
+            "design, all requests should answer quite quickly, in theory a few seconds or less"
+        ),
+    )
+    # TODO: this should not work with a one only request right?
```

Comment:
@sanderegg what do you mean here?

Reply:
what I meant is that you have 2 different time-outs. And the problem we had is that you needed the long one when downloading stuff. We should move to short requests, polling with short requests

Reply:
The `api_request_timeout` is the only timeout used for the dynamic-sidecar requests.
The below `timeout_fetch_dynamic_sidecar_node_id` Is used by an aiodocker call. Not an http call.

Weill remove you comment from the code and expand the description of the `timeout_fetch_dynamic_sidecar_node_id ` field